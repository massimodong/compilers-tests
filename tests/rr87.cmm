struct{
}
la0i20Q;
struct{
}
;
int Q1Pk1g(){
	int gEqe8V[01620], RV_dKis = ZXrTDOs(), qNl7bHHxZ[117][0x8D] = 67.66, GZx6pe[4617] = 020217;
	struct{
	}
	VAUoe, F3pJb;
	int TbLaFH6K, eAXkhWn, FbTs1Tuv[01334347][0xabd], mfMSRbH;
	float w5gy_vWDF;
	0x65F;
}

