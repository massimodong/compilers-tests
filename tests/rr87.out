Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        StructSpecifier (1)
          STRUCT
          LC
          RC
      ExtDecList (3)
        VarDec (3)
          ID: la0i20Q
      SEMI
    ExtDefList (4)
      ExtDef (4)
        Specifier (4)
          StructSpecifier (4)
            STRUCT
            LC
            RC
        SEMI
      ExtDefList (7)
        ExtDef (7)
          Specifier (7)
            TYPE: int
          FunDec (7)
            ID: Q1Pk1g
            LP
            RP
          CompSt (7)
            LC
            DefList (8)
              Def (8)
                Specifier (8)
                  TYPE: int
                DecList (8)
                  Dec (8)
                    VarDec (8)
                      VarDec (8)
                        ID: gEqe8V
                      LB
                      INT: 912
                      RB
                  COMMA
                  DecList (8)
                    Dec (8)
                      VarDec (8)
                        ID: RV_dKis
                      ASSIGNOP
                      Exp (8)
                        ID: ZXrTDOs
                        LP
                        RP
                    COMMA
                    DecList (8)
                      Dec (8)
                        VarDec (8)
                          VarDec (8)
                            VarDec (8)
                              ID: qNl7bHHxZ
                            LB
                            INT: 117
                            RB
                          LB
                          INT: 141
                          RB
                        ASSIGNOP
                        Exp (8)
                          FLOAT: 67.660004
                      COMMA
                      DecList (8)
                        Dec (8)
                          VarDec (8)
                            VarDec (8)
                              ID: GZx6pe
                            LB
                            INT: 4617
                            RB
                          ASSIGNOP
                          Exp (8)
                            INT: 8335
                SEMI
              DefList (9)
                Def (9)
                  Specifier (9)
                    StructSpecifier (9)
                      STRUCT
                      LC
                      RC
                  DecList (11)
                    Dec (11)
                      VarDec (11)
                        ID: VAUoe
                    COMMA
                    DecList (11)
                      Dec (11)
                        VarDec (11)
                          ID: F3pJb
                  SEMI
                DefList (12)
                  Def (12)
                    Specifier (12)
                      TYPE: int
                    DecList (12)
                      Dec (12)
                        VarDec (12)
                          ID: TbLaFH6K
                      COMMA
                      DecList (12)
                        Dec (12)
                          VarDec (12)
                            ID: eAXkhWn
                        COMMA
                        DecList (12)
                          Dec (12)
                            VarDec (12)
                              VarDec (12)
                                VarDec (12)
                                  ID: FbTs1Tuv
                                LB
                                INT: 375015
                                RB
                              LB
                              INT: 2749
                              RB
                          COMMA
                          DecList (12)
                            Dec (12)
                              VarDec (12)
                                ID: mfMSRbH
                    SEMI
                  DefList (13)
                    Def (13)
                      Specifier (13)
                        TYPE: float
                      DecList (13)
                        Dec (13)
                          VarDec (13)
                            ID: w5gy_vWDF
                      SEMI
            StmtList (14)
              Stmt (14)
                Exp (14)
                  INT: 1631
                SEMI
            RC
